src/app/f2f/controllers/abort.js

```diff
@@ -0,0 +1,27 @@
+const BaseController = require("hmpo-form-wizard").Controller;
+const { API } = require("../../../lib/config");
+
+class AbortController extends BaseController {
+  next() {
+    return '/done'
+  }
+  async saveValues(req, res, callback) {
+    try {
+      await this.abortJourney(req);
+      callback();
+    } catch (err) {
+      callback(err);
+    }
+  }
+  async abortJourney(req) {
+    const headers = {
+      "x-govuk-signin-session-id": req.session.tokenId
+    };
+
+		const resp = await req.axios.post(`${API.PATHS.ABORT}`, {}, {
+      headers,
+    });
```

Comment:
there is no try/catch around this. When the POST request returns 4XX or 5XX this results in an unhandled exception causing the whole container instance to crash, and ECS has to destroy it and start up a new container instance

Reply:
Will it not be covered by the catch block on line 12 above?